action.yml

```diff
@@ -0,0 +1,47 @@
+name: "cloud.gov deploy"
```

Comment:
[question] how does this file work with the `/github/workflows/actions.yml` file?

Reply:
This file is the shared action that will be `uses`ed by other workflows. `.github/workflows/action.yml` is the local workflow that tests the shared one.